packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts

```diff
@@ -113,12 +115,12 @@ export class PaginatedMessage {
 	/**
 	 * The response message used to edit on page changes.
 	 */
-	public response: Message | null = null;
+	public response: Message | CommandInteraction | SelectMenuInteraction | ButtonInteraction | null = null;
 
 	/**
 	 * The collector used for handling button clicks.
 	 */
-	public collector: InteractionCollector<MessageComponentInteraction> | null = null;
```

Comment:
Why's the type changed again?

Reply:
Before the collector was created through 
```ts
			this.collector = this.response!.createMessageComponentCollector({
				filter: (interaction) => this.actions.has(interaction.customId),
				idle: this.idle
			})
```
 however there is no `createMessageComponentCollector` for any of the `*Interaction` classes so now it's created through 
```ts
			this.collector = new InteractionCollector(targetUser.client, {
				filter: (interaction) => interaction.isMessageComponent() && this.actions.has(interaction.customId),
				time: this.idle,
				guild: isGuildBasedChannel(channel) ? channel.guild : undefined,
				channel,
				interactionType: Constants.InteractionTypes.MESSAGE_COMPONENT
			})
```
which inadvertently also changed the type

Reply:
Its still an InteractionCollector that collects MessageComponentInteraction